mm/shm/shmat.c

```diff
@@ -237,10 +238,10 @@ FAR void *shmat(int shmid, FAR const void *shmaddr, int shmflg)
 
   /* Set aside a virtual address space to span this physical region */
 
-  vaddr = shm_alloc(group, NULL, region->sr_ds.shm_segsz);
+  vaddr = vpage_alloc(get_group_mm(group), NULL, region->sr_ds.shm_segsz);
```

Comment:
I think the name `vpage_alloc()` is not quite right. Consider something like `vm_alloc()` or maybe `vm_alloc_region()` ?

Reply:
vm_alloc_region sounds good to me

Reply:
done